src/stories/Library/disclosure/Disclosure.tsx

```diff
@@ -9,6 +10,7 @@ export type DisclosureProps = {
   withAvailability?: boolean;
   fullWidth?: boolean;
   removeHeadlinePadding?: boolean;
+  headingLevel?: HeadingLevelType;
```

Comment:
Please make the heading level required.

[The argumentation is the same as for the React components](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/387#discussion_r1135104177).